app/src/main/java/com/codesoom/assignment/dto/ProductResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.codesoom.assignment.dto;
+
+import com.codesoom.assignment.domain.Product;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Builder
+@Getter
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class ProductResponse {
+
+    private Long id;
+
+    private String name;
+
+    private String maker;
+
+    private int price;
+
+    private String imageUrl;
+
+    public static ProductResponse of(Product product) {
+        return ProductResponse.builder()
+                .id(product.getId())
+                .name(product.getName())
+                .maker(product.getMaker())
+                .price(product.getPrice())
+                .imageUrl(product.getImageUrl())
+                .build();
+    }
+
+    public static List<ProductResponse> listOf(List<Product> products) {
```

Comment:
이렇게 할 수도 있을까요? 할 수 있다면 어떤 장점이 있을까요?
```suggestion
    public static List<ProductResponse> listOf(Collection<Product> products) {
```

Reply:
상품들이 저장된 컬렉션에 상관없이 해당 메서드를 사용할 수 있게 되어 훨씬 더 유연한 메서드가 될거 같습니다!!